src/server/editorServices.ts

```diff
@@ -1377,6 +1378,9 @@ namespace ts.server {
                 );
 
             project.addRoot(info);
+            if(this.hostConfiguration.typeAcquisition){

```

Comment:
This is not the place to set it at all..  It should be set when inferred project is created instead.

Reply:
How would that interact with changing the setting after the project is created? Does/should a new project get created everytime the setting is changed?

Reply:
You would set it for inferred project and mark them dirty.